features/support/env.rb

```diff
@@ -1,6 +1,9 @@
 require 'aruba/cucumber'
 
 Before do
+  # Force ids to be printed unquoted for consistency
+  set_env('SHELL', '/usr/bin/bash')
```

Comment:
As mentioned in my comment below, we chose to not quote example ids on shells which do not require it (bash, for example), which is a better experience for the user, IMO.

However, to ensure consistency in output within our spec suite, so that our specs consistently pass whether run in zsh, bash or some other shell, we fake it out here and pretend the shell is bash so that the IDs are not quoted within our suite.